components/Main.tsx

```diff
@@ -1,50 +1,51 @@
-/* eslint-disable react/no-unescaped-entities */
 import React from "react";
 import { AiOutlineMail } from "react-icons/ai";
 import { FaGithub, FaLinkedinIn } from "react-icons/fa";
 import { BsFillPersonLinesFill } from "react-icons/bs";
 import Link from "next/link";
-import Navbar from "./Navbar";
+import Navbar from "./NavBar/Navbar";
```

Comment:
It’ll better to have index and export everything there

Reply:
iam exporting every thing in app.js